.github/workflows/kuksa_databroker_build.yml

```diff
@@ -173,6 +176,10 @@ jobs:
       run: |
         ./scripts/build-databroker.sh ${{ matrix.platform.name }}
 
+    - name: "Check each feature"
+      working-directory: ${{github.workspace}}
+      run: cargo hack check --each-feature
```

Comment:
It seems we run this for each platform in the build matrix, but I think we do nothing platform specific here, so maybe moving it out can save some time?

Reply:
Maybe in the (unit) test job?